Cargo.toml

```diff
@@ -14,3 +14,4 @@ exclude = [
 
 [workspace.lints.rust]
 rust_2018_idioms = "warn"
+unused_qualifications = "warn"
```

Comment:
Github is complaining about the lack of trailing new line. Not that it particularly matters, unless you're a POSIX purist 😆.

Reply:
Fixed! 🤷